### PR TITLE
支持透明背景微信小程序码

### PR DESCRIPTION
**Pull request by oldcai**

### CODE REVIEW

File: wechatpy/client/api/wxa.py [hunk 1]
```diff
@@ -44,20 +50,22 @@ def get_wxa_code_unlimited(self,
...
+                               page=None,
+                               is_hyaline=False):
         """
         创建小程序码（接口B：适用于需要的码数量极多，或仅临时使用的业务场景）
         详情请参考
         https://mp.weixin.qq.com/debug/wxadoc/dev/api/qrcode.html
         """
         return self._post(
             'wxa/getwxacodeunlimit',
-            data=optionaldict(
```
Comment (messense) [inline wechatpy/client/api/wxa.py]: I'd prefer not to change this.

Comment (oldcai) [reply wechatpy/client/api/wxa.py]: Could you give me a reason to use `optionaldict` instead of `dict` in this particular place?

Comment (messense) [reply wechatpy/client/api/wxa.py]: To avoid having `{"page": null}` in JSON.

Comment (oldcai) [reply wechatpy/client/api/wxa.py]: That makes sense.

**Review by messense — APPROVED** [review]

**Pull request merged**